templates/haskell/package.yaml

```diff
@@ -0,0 +1,139 @@
+# Use this file as a reference when you create package.yaml
+# for your new package.
+# If you have multiple packages, you may wish to extract common
+# definitions into a separate file.
+# Why we need this template:
+# • harmless default extensions
+# • a good set of warnings
+# • metadata fields that are better be set
+# • some tricks (e. g. RTS options)
+name:                myproject
+# version is 0 by default to indicate that nothing is ready
+version:             0
+# adjust/add more authors and maintainers if necessary
+author:              Serokell
+maintainer:          Serokell <hi@serokell.io>
+# other metadata fields we usually set:
+# • copyright
+# • license, license-file
+# • git
+# • homepage
+# • bug-reports
+
```

Comment:
We can also include synopsis, category, and description fields here, I think those are pretty standard. If we include them with dummy values, then newcomers will know that they are expected to include those fields.